example/python/cov.xml

```diff
@@ -3,7 +3,7 @@
 	<!-- Generated by coverage.py: https://coverage.readthedocs.io -->
 	<!-- Based on https://raw.githubusercontent.com/cobertura/web/master/htdocs/xml/coverage-04.dtd -->
 	<sources>
-		<source>/home/ryanluker/dev/vscode-coverage-gutters/example/python/foobar</source>
+		<source>/home/brandongk/Desktop/vscode-coverage-gutters/example/python/foobar</source>
```

Comment:
@ryanluker -> We need to make this file auto-calculate as well, it will fail anywhere except my machine.